text/0060-tasks-api.md

```diff
@@ -103,6 +103,7 @@ List of global tasks by `type`:
 | name                | description                          |
 |---------------------|--------------------------------------|
 | providedIds         | Number of provided document ids.     |
+| originalFilter      | The filter used to delete documents. |
```

Comment:
This field is only visible when deleting tasks with `POST` - `indexes/:index_uid/documents/delete`, right?

Reply:
Hey yes! I'll add it to the spec!